packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -256,6 +250,22 @@ export async function copyProjectTemplateAndReplace(
     });
   }
 
+  const packagesConfigCppNugetPackages = [
+    ...cppNugetPackages,
+    {
+      id: 'Microsoft.UI.Xaml',
```

Comment:
If this is no longer conditional, does this cause a large memory increase because WinRT tools will now merge multiple .winmd files?

Reply:
this is fixed in the other PR (#7014)  so it will be addressed once that one is checked in and I rebase :)

Reply:
update: rebased it now, i'm hoping to check 7014 in soon 🤞